docs/content/integrations/dbt/reference.mdx

```diff
@@ -124,7 +124,7 @@ dagster dev
 In production, a precompiled manifest should be used. Using <PyObject object="DbtProject" module="dagster_dbt" />, the manifest can be created at build time by running the [`dagster-dbt project prepare-and-package`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-prepare-and-package) command in your CI/CD workflow. For more information, see the [Deploying a Dagster project with a dbt project](#deploying-a-dagster-project-with-a-dbt-project) section.
 
 </TabItem>
-<TabItem name="Option 2: Using DbtCliResource">
+<TabItem name="Option 2 (legacy): Using DbtCliResource">
```

Comment:
Keeping this section for users using the `DbtCliResource` strategy in their code. 

I'm open to remove it and share a previous version of the docs to users if they have questions.

Thoughts?

Reply:
We should get rid of it so we're not promoting two ways of doing the same thing. Also, this will steer more folks to using `DbtProject` to battle test it.

Reply:
Updated in [9f73d9f](https://github.com/dagster-io/dagster/pull/23263/commits/9f73d9f4ca8ef2149cf5b825597e3cf4ae714b37). I've also updated the "Deploying a Dagster project with a dbt project" section to remove `DbtCliResource` as an option.